[2024-12-31T10:44:56.617+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-31T10:44:56.653+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: final_backup1.initialize_data scheduled__2024-12-30T00:00:00+00:00 [queued]>
[2024-12-31T10:44:56.668+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: final_backup1.initialize_data scheduled__2024-12-30T00:00:00+00:00 [queued]>
[2024-12-31T10:44:56.669+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 4
[2024-12-31T10:44:56.689+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): initialize_data> on 2024-12-30 00:00:00+00:00
[2024-12-31T10:44:56.702+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=209) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-31T10:44:56.703+0000] {standard_task_runner.py:72} INFO - Started process 224 to run task
[2024-12-31T10:44:56.700+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'final_backup1', 'initialize_data', 'scheduled__2024-12-30T00:00:00+00:00', '--job-id', '1045', '--raw', '--subdir', 'DAGS_FOLDER/final_backup1.py', '--cfg-path', '/tmp/tmp5bnz2tu6']
[2024-12-31T10:44:56.705+0000] {standard_task_runner.py:105} INFO - Job 1045: Subtask initialize_data
[2024-12-31T10:44:56.731+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-12-31T10:44:56.792+0000] {task_command.py:467} INFO - Running <TaskInstance: final_backup1.initialize_data scheduled__2024-12-30T00:00:00+00:00 [running]> on host 05c0118fb254
[2024-12-31T10:44:56.795+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-12-31T10:44:56.955+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='final_backup1' AIRFLOW_CTX_TASK_ID='initialize_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-30T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-30T00:00:00+00:00'
[2024-12-31T10:44:56.958+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-31T10:44:56.962+0000] {final_backup1.py:117} INFO - Initializing workflow state...
[2024-12-31T10:44:56.964+0000] {final_backup1.py:119} INFO - Workflow initialized with state: {'workflow_state': 'initialized', 'files_to_copy': {}}
[2024-12-31T10:44:56.965+0000] {python.py:240} INFO - Done. Returned value was: {'workflow_state': 'initialized', 'files_to_copy': {}}
[2024-12-31T10:44:57.000+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-31T10:44:57.001+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=final_backup1, task_id=initialize_data, run_id=scheduled__2024-12-30T00:00:00+00:00, execution_date=20241230T000000, start_date=20241231T104456, end_date=20241231T104457
[2024-12-31T10:44:57.043+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-31T10:44:57.114+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-31T10:44:57.117+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
